azure-pipelines.yml

```diff
@@ -49,8 +49,12 @@ jobs:
         displayName: Add relavent Channels
 
       - bash: |
-          conda create -n test-environment python=$(python.version) --file requirements-dev.txt
+          conda update -c defaults conda
+          conda update --all
+          conda create -n test-environment
           source activate test-environment
+          conda install --yes python=$(python.version) --file requirements-dev.txt
```

Comment:
Will this add both python versions?

Reply:
It looks like the correct python versions are being installed in the latest build: https://dev.azure.com/mosdef/mosdef/_build/results?buildId=135 (see the "Run Tests" step).